src/Illuminate/Database/Migrations/MigrationCreator.php

```diff
@@ -184,12 +184,13 @@ protected function getPath($name, $path)
      * Fire the registered post create hooks.
      *
      * @param  string|null  $table
+     * @param  string       $path
      * @return void
      */
-    protected function firePostCreateHooks($table)
+    protected function firePostCreateHooks($table, $path)
```

Comment:
This is a breaking change to the method signature.

Reply:
Thank you @driesvints . Should I request this merge request to another branch? To `master`?

Reply:
Yes best to master.